### PR TITLE
Issue #147: Add default view for Who's New block.

### PR DESCRIPTION
**Pull request by ghost**
Adds default view for: https://github.com/backdrop/backdrop-issues/issues/147

### CODE REVIEW

File: core/modules/user/config/views.view.user_new.json [hunk 1]
```diff
@@ -0,0 +1,146 @@
+{
+  "_config_name": "views.view.user_new",
+  "name": "user_new",
+  "description": "Lists new users who have accessed the site successfully.",
+  "tag": "default",
+  "disabled": false,
+  "base_table": "users",
```
Comment (quicksketch) [inline core/modules/user/config/views.view.user_new.json:8]: One thing we can add here that will enable the "revert" ability is if we add `"module": "user"`. Then the view will have the ability to be enabled/disabled/reverted, but not deleted.

**Comment by jenlampton** [comment]
Marking NW because this still needs an upgrade path.  I added some docs to the meta issue about how to approach upgrade paths. https://github.com/backdrop/backdrop-issues/issues/151

**Comment by ghost** [comment]
Untested upgrade path commit.

**Comment by ghost** [comment]
Closing as per https://github.com/backdrop/backdrop-issues/issues/147

**Comment by quicksketch** [comment]
Thanks @BWPanda!

**Pull request closed without merging**